example/src/modules/example.js

```diff
@@ -78,6 +78,14 @@ export default class Example extends Component<Props, State> {
     }
   };
 
+  getResponseRecaptcha = () => {
+    if (this.captcha) {
+      this.captcha.getResponse().then(response => {
+        alert(response);
+      });
```

Comment:
@sarneeh 
The google `getResponse` API will return just an `""` empty string until the challenge is resolved.

I'm just curious if it should be treated as a Promise rejection 🤔 
Current implementation would only reject the Promise if the underlying Google API would `throw`, which I consider valid behaviour too.